### DISCUSSION TITLE
Pymoo: individuals

### DISCUSSION THREAD

**Discussion by RafaHPSUnicamp** [anyoptimization/pymoo#504] [Q&A]
I am using Pymoo and have a simple question. How individual is represented by both Pymoo and NSGA-II/C-TAEA/NSGA-III.

I have learned that population is selected through sampled (can be random), using all possibilities of output variables. But how the individual can be represented in this case?

It is for studying. I will accept direct links and texts with a direct answer for me.

Resume: have a question about how popluation and individuals can be explained or represented in any problem of Pymoo.

**Comment by blankjul** [comment]
You can checkout the core classes in pymoo here: 
https://github.com/anyoptimization/pymoo/tree/main/pymoo/core

 A `Population` is a collection of `Individual`. An `Individual` contains the design, objective, and constrained space information. 
The `Sampling` operators creates a new `Population`.
